conda/generate_recipe.sh

```diff
@@ -0,0 +1,141 @@
+    fi
+    git -C "$REPO" clean -xfd &>/dev/null
+    git -C "$REPO" submodule foreach --recursive git clean -xfd &>/dev/null
+    git -C "$SCRIPT_DIR" clean -xfd &> /dev/null
+}
+
+function string_replace()
+{
+    printf "Creating recipes at $TMP_DIR/$RECIPES\n"
+    REPLACE=(APPLICATION FORMATTED_APPLICATION EXECUTABLE REPO BUILD VERSION MOOSE_JOBS MOOSE IS_MOOSE TMP_DIR RECIPES SKIP_DOCS SUBMODULES EXTRA_SUBMODULES PREFIX_PACKAGE_WITH)
+    for sfile in `find $SCRIPT_DIR/$TEMPLATE -type l`; do
+        cat "$sfile" > "$TMP_DIR/$RECIPES/$(basename $sfile)"
+    done
+    for cfile in `find $SCRIPT_DIR/$TEMPLATE -type f`; do
+        cp "$cfile" "$TMP_DIR/$RECIPES"
+        for THIS in "${REPLACE[@]}"; do
+            if [ `uname ` == "Darwin" ]; then
+                sed -i '' -e "s|\<${THIS}\>|${!THIS}|g" "$TMP_DIR/$RECIPES/$(basename $cfile)"
+            else
+                sed -i'' -e "s|<${THIS}>|${!THIS}|g" "$TMP_DIR/$RECIPES/$(basename $cfile)"
+            fi
+        done
+    done
+}
+
+function conda_build()
+{
+    if [ -z "$CIVET_HOME" ]; then
+        printf "Installing everything necessary to a temporary location (including Conda)\n"
+        create_tmp
+        printf "TEMP DIR: $TMP_DIR\n"
+        create_env
+        clean_repo
+        string_replace || exit 1
+        cd "$TMP_DIR/$RECIPES" || exit 1
+        mkdir -p "${SCRIPT_DIR}/packages/${APPLICATION}" || exit 1
+        conda mambabuild . --output-folder "${SCRIPT_DIR}/packages/${APPLICATION}" || exit 1
+        printf "Built: ${SCRIPT_DIR}/packages/${APPLICATION}/${ARCH}/${PREFIX_PACKAGE_WITH}${FORMATTED_APPLICATION}-${VERSION}-build_${BUILD}.tar.bz2\n"
+    else
+        TMP_DIR="$BUILD_ROOT"
+        clean_repo
+        create_tmp || exit 1
+        string_replace || exit 1
+    fi
+}
+
+if ! `type conda &>/dev/null` || ! `type activate &>/dev/null`; then
+    printf "Conda not installed. Or not properly available (activate not available).\n"
+    exit 1
+elif [ -z "$2" ]; then
+    printf "Supply possitional arguments:\n\t$0 /path/to/application jobs\n\nExample:\n\t$0 /home/user/moose 24\n"
+    exit 1
+elif ! [ -d "$1" ]; then
+    printf "$1 not found\n"
+    exit 1
+fi
+if [ "$(uname)" == "Darwin" ]; then
+    ARCH="osx-64"
+else
+    ARCH="linux-64"
+fi
+# We are building moose
+if [ "$(basename "$REPO" .git)" == "moose" ]; then
+    export IS_MOOSE='/modules'
+    export MOOSE=''
+    export EXECUTABLE='combined'
+    export PREFIX_PACKAGE_WITH=''
+    export MOOSE_SKIP_DOCS='true'
+else
+    export IS_MOOSE=''
+    export MOOSE='/moose'
+    export PREFIX_PACKAGE_WITH=${PREFIX_PACKAGE_WITH:-'ncrc-'}
+fi
+SKIP_DOCS='False'
+if [ -n "$MOOSE_SKIP_DOCS" ]; then
+    printf "Influential environment variable: MOOSE_SKIP_DOCS detected.\n"
+    export SKIP_DOCS="True"
+fi
+SUBMODULES="False"
+if [ -n "$EXTRA_SUBMODULES" ]; then
+    printf "Influential environment variable: EXTRA_SUBMODULES detected.\n"
+    export SUBMODULES="True"
+fi
```

Comment:
remove this logic. Submodules are being handled by Civet recipes during fetch_and_branch.sh